docs/streams/stream-basics/consumers.md

```diff
@@ -38,3 +38,60 @@ Client libraries can provide their own listener implementations for consumers. I
 ## Cursor
 
 A cursor is the subscription position for a stream consumer. You can think of it like the cursor on your page, marking where you are.
+
+## Create Consumers
+
+It is possible to use our JavaScript or Python SDK to create producers.
+
+<Tabs groupId="modify-single">
+<TabItem value="javascript" label=" JavaScript SDK">
+
+- Step 1. [Install the SDK](../../sdks/install-sdks.md).
+- Step 2. Change `BASE_URL` if necessary and insert `apiKey`.
+- Step 3. Create an instance of the jsC8.
+- Step 4. Request `stream` object.
+- Step 5. Request One Time Password and create consumer.
+
+```js
+const jsc8 = require("jsc8");
+
+const BASE_URL = "https://play.paas.macrometa.io/"
+
+client = new jsc8({
+    url: BASE_URL,
+    apiKey: "xxxxxx",
+    fabricName: "_system",
+});
+
+const streamName = "streamQuickstart";
+const subscriptionName = "consumer-subscription"
+
+async function consumer() {
+  try {
+    await console.log("\nConnecting consumer to global stream...");
```

Comment:
@dlozina-macrometa should we have another variable isLocal to specify if stream is global or local and use that instead of hardcoding it to global?? Also if we do this then the Url for consumer (dcName) will need to have additional logic .. if isLocal use regional url else use base url.

Reply:
Also Whatever changes we make here is automatically applicable to all examples in subscriptions folder since we are using the same consumer code there.

Reply:
I might create a whole another example for local stream. There will be to much information in the example if we add local/global selection.
Goal is to write simple "bare bones" stuff people can just copy and understand at first glance. I always imagine a new developer that just wants to run the code, check out if it works, then dig deeper. What do you think? @koshy-thomas

Reply:
@dlozina-macrometa ... That also works as long as we have both local and global streams covered (in separate example or same example both way works :) ).

Reply:
But there is one more thing if we create separate example.. then it would be required in all the sub code examples in subscriptions folder, producer, consumer, etc.. However, you think is the best way to cover both local and global stream I am onboard with it as long both type of streams are covered :)

Reply:
Agreed 💯 !